CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ Mapbox welcomes participation and contributions from everyone.
 * Fix for location indicator not being rendered at the horizon when terrain is enabled. ([#1351](https://github.com/mapbox/mapbox-maps-android/pull/1351))
 * Fix for loading gltf models with interleaved buffers. ([#1351](https://github.com/mapbox/mapbox-maps-android/pull/1351))
 * Fix a bug that `line-trim-offset` input may lose precision via shader calculation. ([#1359](https://github.com/mapbox/mapbox-maps-android/pull/1359))
+* Fix rethrowing pending java exceptions on the main thread to avoid a native crash. ([#1363](https://github.com/mapbox/mapbox-maps-android/pull/1363))
```

Comment:
```suggestion
* Rethrowing pending java exceptions on the main thread to avoid a native crash. ([#1363](https://github.com/mapbox/mapbox-maps-android/pull/1363))
```

Reply:
it does not change the thread if I understand that correctly
I'd go with following and also move this from bug fix section to `Features and improvements`
```
Re-throw native exceptions `jni::PendingJavaException` as readable Java exceptions with detailed exception text.
```